docs/databags.md

```diff
@@ -16,9 +16,13 @@ The Cluster Hash is part of a plaintext data bag item that defines a logical gro
 
 * `['license_uri']` - SplunkAPI URI of the License Server (Required for getting onto the Enterprise license, if unset, use trial license)
 * `['master_uri']` - SplunkAPI URI of the cluster master (Required for servers connecting to managed clusters)
+* `['deployer_uri']` - SplunkAPI URI of the Deployer in the Search Head Cluster
+* `['shc_members']` - Array of Splunk API URIs of the search head members that are in the SH (Search Head) cluster. Always make sure that the first SH member in the array is an existing member in the cluster.
```

Comment:
Minor, I'm guessing the second sentence here could be qualified with "When adding members to an existing cluster, always make sure ..."  Otherwise it would be impossible to setup a new cluster.